namespace cpp facebook.fboss.thrift
namespace cpp2 facebook.fboss.thrift
namespace py neteng.fboss
namespace d neteng.fboss

exception FbossBaseError {
  1: string message
} ( message = "message" )
